site_config/community.jsx

```diff
@@ -40,6 +40,64 @@ export default {
           }
         ],
       },
+      {
```

Comment:
There are 14 documents migrated from `docs` to `community`, but only 12 documents were referenced